utils/mockDGCClient.ts

```diff
@@ -0,0 +1,27 @@
+import * as E from "fp-ts/Either";
```

Comment:
Besides what literature says, I would just call this file "fake" as the term _mock_ is used in our codebase to refer dummy, test implementations.

Reply:
Done in [583aef4](https://github.com/pagopa/io-functions-eucovidcerts/pull/76/commits/583aef4d787e6354b264367c2cbfa667e512f2b6)